drill.cpp

```diff
@@ -485,12 +501,15 @@ bool ExcellonProcessor::millhole(std::ofstream &of, double start_x, double start
 
         // Start one step above Z0 for optimal entry
         of << "G1 Z" << -1.0/stepcount * cutter->zwork * cfactor
-           << " F" << cutter->vertfeed * cfactor << '\n';
+           << " F" << cutter->g0_horizontal_speed * cfactor << '\n';
```

Comment:
This should be remain `vertfeed`, no?  Why this change?

I noticed that the code is "G1" but the variable is called "g0".

Reply:
I think this is moving down from zsafe to the top of the work piece. In my mind this motion can be as fast as possible since no cutting should occur. Maybe I'm wrong about this, I'll change it back. The time difference for most simple cartesian setups is only going to be milliseconds anyways